src/components/ui/AlertsSection/index.tsx

```diff
@@ -1,29 +1,36 @@
 import React from 'react';
-import PageSection from '../PageSection';
 import { AlertGroup, AlertActionCloseButton, Alert } from '@patternfly/react-core';
+import PageSection from '../PageSection';
 import { AlertProps } from '../../../features/alerts/alertsSlice';
+import { AlertsContext } from '../../AlertsContextProvider';
 
 import './AlertsSection.css';
-import { AlertsContext } from '../../AlertsContextProvider';
 
-const AlertsSection: React.FC = () => {
+export const AlertsSectionGroup: React.FC<{ className?: string }> = ({ className }) => {
```

Comment:
```suggestion
export const Alerts: React.FC<{ className?: string }> = ({ className }) => {
```
Since this has nothing to do with the page section how about using this name?